addons/addon-base-raas/packages/base-raas-services/lib/alb/__tests__/alb-service.test.js

```diff
@@ -430,4 +430,63 @@ describe('ALBService', () => {
       }
     });
   });
+
+  describe('modifyRule', () => {
+    it('should pass and return empty object with success', async () => {
+      service.getHostname = jest.fn(() => {
+        return 'rtsudio-test.example.com';
+      });
+      service.findAwsAccountDetails = jest.fn(() => {
+        return {
+          externalId: 'subnet-0a661d9f417ecff3f',
+        };
+      });
+      albClient.modifyRule = jest.fn().mockImplementation(() => {
+        return {
+          promise: () => {
+            return {};
+          },
+        };
+      });
+      service.getAlbSdk = jest.fn().mockResolvedValue(albClient);
+      const response = await service.modifyRule({}, { cidr: [], projectId: '' });
+      expect(response).toEqual({});
+    });
+    it('should pass when user passed empty cidr value to modify rule', async () => {
+      // the system should validate and replace the default ip "0.0.0.0/0" and execute
+      albClient.modifyRule = jest.fn().mockImplementation(() => {
+        return {
+          promise: () => {
+            return {};
+          },
+        };
+      });
+      service.getAlbSdk = jest.fn().mockResolvedValue(albClient);
+      await service.modifyRule({}, { cidr: [], projectId: '' });
+      expect(albClient.modifyRule).toHaveBeenCalled();
+    });
+  });
+
+  describe('describeRules', () => {
```

Comment:
Can we test non-empty response use case as well

Reply:
Yes. Added non-empty response use case as well. Please review